extensions/notebook/src/jupyter/localCondaPackageManageProvider.ts

```diff
@@ -104,7 +104,20 @@ export class LocalCondaPackageManageProvider implements IPackageManageProvider {
 
 				let packages = packageJson[packageName];
 				if (Array.isArray(packages)) {
-					let allVersions = packages.filter(pkg => pkg && pkg.version).map(pkg => pkg.version);
+					let allVersions = packages.filter(pkg => {
+						if (pkg && pkg.version) {
+							let dependencies = pkg.depends;
+							if (Array.isArray(dependencies)) {
+								let strDependencies = dependencies as string[];
+								let pythonDependency = strDependencies.find(dependency => dependency.trim().toLowerCase().startsWith('python '));
```

Comment:
Is this going to always be guaranteed to find a value? Might want to do `pythonDependency?.replace` in the next line for safety